lib/jellyfish_web/socket.ex

```diff
@@ -33,46 +37,97 @@ defmodule JellyfishWeb.Socket do
 
       {:ok, state}
     else
-      {:ok, :connected} ->
+      params: :error ->
+        Logger.warn(
+          "No room_id or peer_id in connection params, ignoring incoming WebSocket connection"
+        )
+
+        {:error, :no_params}
+
+      peer: {:ok, _status} ->
         Logger.warn("""
         WebSocket connection for peer #{inspect(state.params["peer_id"])} in room \
         #{inspect(state.params["room_id"])} already exists, rejected
         """)
 
         {:error, :already_connected}
```

Comment:
Won't we also go through this case if peer status is `:authenticating`? If yes I think the log should be adjusted.